SWIPs/swip-honey_to_money.md

```diff
@@ -42,10 +42,10 @@ In the current implementation, honey prices are hardcoded. When this price in co
 
```

Comment:
"Upon receiving a payment. nodes will determine the honey amount paid by looking up the currently valid price from the oracle and divide the received currency amount by this value" or something like that.

Reply:
Good point. Will include